contracts/core/src/contract.rs

```diff
@@ -843,6 +843,16 @@ fn execute_tick_claiming(
         }
         drop_puppeteer_base::msg::ResponseHookMsg::Error(err) => {
             attrs.push(attr("error_on_claiming", format!("{:?}", err)));
+            match err.transaction {
+                drop_puppeteer_base::msg::Transaction::ClaimRewardsAndOptionalyTransfer {
+                    ..
+                } => {
+                    FSM.go_to(deps.storage, ContractState::Idle)?;
```

Comment:
I believe here we should also add moving the batch back to Unbonding status, or it will stuck in the Withdrawing status. Also, it means that we need to introduce a new knot for that. Can we also please extend the test to be sure that this batch will be picked up during the next loop run?

Reply:
okay 👍 addressed